src/components/MultiHopTrade/components/LimitOrder/LimitOrderList.tsx

```diff
@@ -56,48 +69,60 @@ export const LimitOrderList: FC<LimitOrderListProps> = ({ cardProps }) => {
 
   return (
     <Card {...cardProps}>
-      <CardHeader px={0} pt={4} h='full'>
-        <Tabs variant='unstyled' display='flex' flexDirection='column' h='full'>
-          <TabList gap={4} flex='0 0 auto' mb={2} ml={6}>
-            <Tab
-              p={0}
-              fontSize='md'
-              fontWeight='bold'
-              color='text.subtle'
-              _selected={textColorBaseProps}
-            >
-              <Text translation='limitOrders.openOrders' />
-            </Tab>
-            <Tab
-              p={0}
-              fontSize='md'
-              fontWeight='bold'
-              color='text.subtle'
-              _selected={textColorBaseProps}
-            >
-              <Text translation='limitOrders.orderHistory' />
-            </Tab>
-          </TabList>
+      {onBack && (
+        <CardHeader px={0} display='flex' flexDirection='column' pb={0}>
+          <Flex width='full' alignItems='center' mb={4} position='relative' mx={4}>
+            <Box position='absolute' left={0}>
```

Comment:
preferably-blocking: avoid the position absolute which could cause z-index/hover issues and where the back button could not be clickable because of siblings elements

Reply:
ty, removed absolute position in https://github.com/shapeshift/web/pull/8073/commits/2299b23e8f81aeada33843d518e4aa4bfdfdf786